### PR TITLE
LPD-40087 - Technical Task | Create Workflow Definition Links endpoint by ERC and ID

### PR DESCRIPTION
**Pull request by JhossephG**
This is the final PR with all the needs to implement the features from story [LPD-36879](https://liferay.atlassian.net/browse/LPD-36879).

PR of the first task [here](https://github.com/brianchandotcom/liferay-portal/pull/155702)

PR of the second task [here](https://github.com/liferay-bpm/liferay-portal/pull/3780)

[LPD-36879]: https://liferay.atlassian.net/browse/LPD-36879?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by JhossephG** [comment]
ci:test:sf

**Comment by JhossephG** [comment]
ci:test:relevant

**Comment by JhossephG** [comment]
ci:test:workflow

**Comment by JhossephG** [comment]
ci:test:relevant

**Review by linolaoj — CHANGES_REQUESTED** [review]

File: modules/apps/headless/headless-admin-workflow/headless-admin-workflow-test/src/testIntegration/java/com/liferay/headless/admin/workflow/resource/v1_0/test/WorkflowDefinitionLinkResourceTest.java [hunk 1]
```diff
@@ -6,15 +6,158 @@
...
+@DataGuard(scope = DataGuard.Scope.METHOD)
 @RunWith(Arquillian.class)
 public class WorkflowDefinitionLinkResourceTest
 	extends BaseWorkflowDefinitionLinkResourceTestCase {
+
+	@BeforeClass
+	public static void setUpClass() throws Exception {
+		BaseWorkflowDefinitionLinkResourceTestCase.setUpClass();
+
+		_workflowDefinition =
```
[the remark this answers is not in the document]
Comment (JhossephG) [reply modules/apps/headless/headless-admin-workflow/headless-admin-workflow-test/src/testIntegration/java/com/liferay/headless/admin/workflow/resource/v1_0/test/WorkflowDefinitionLinkResourceTest.java]: done!

**Comment by JhossephG** [comment]
ci:forward:force

**Comment by JhossephG** [comment]
ci:test:stop

**Comment by JhossephG** [comment]
ci:stop

**Comment by JhossephG** [comment]
ci:test:relevant

**Comment by JhossephG** [comment]
ci:stop

**Comment by JhossephG** [comment]
After discuss with @linolaoj we concluded that the errors are not related. Sent directly to BChan [here](https://github.com/brianchandotcom/liferay-portal/pull/155780)

**Pull request closed without merging**